lib/alephant/broker/request/batch.rb

```diff
@@ -10,17 +10,17 @@ class Batch
         attr_reader :batch_id, :components, :load_strategy
 
         def initialize(component_factory, env)
-          logger.info "Request::Batch#initialize: id: #{env.data['batch_id']}"
+          logger.info "Request::Batch#initialize: id: #{env.data['batch_id']}" if env.data
```

Comment:
For the logging level answer, this would come through in the logs on live. I don't think it is environment specific, based on the logging type. I see lots of the info lines in the log, when I have looked in the logs previously.